client/components/logged-out-form/back-link.jsx

```diff
@@ -0,0 +1,66 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { addLocaleToWpcomUrl } from 'lib/i18n-utils';
+import safeProtocolUrl from 'lib/safe-protocol-url';
+import Gridicon from 'gridicons';
+
+function LoggedOutFormBackLink( props ) {
+	const { locale, oauth2Client, translate, recordClick } = props;
+
+	let url = addLocaleToWpcomUrl( 'https://wordpress.com', locale );
+	let message = translate( 'Back to WordPress.com' );
+
+	if ( oauth2Client ) {
+		url = safeProtocolUrl( oauth2Client.url );
+		if ( ! url || url === 'http:' ) {
+			return null;
+		}
+
+		message = translate( 'Back to %(clientTitle)s', {
+			args: {
+				clientTitle: oauth2Client.title,
+			},
+		} );
+	}
+
+	return (
+		<a
+			href={ url }
+			key="return-to-wpcom-link"
+			onClick={ recordClick }
+			rel="external"
+			className={ classnames(
+				Object.assign(
```

Comment:
Can be just
```js
classnames( {
   'logged-out-form__link-item': true,
   'logged-out-form__back-link': true,
   ...props.classes
} )
```
no?

Reply:
Of course, I'll change that. Much tidier!